api/components/examples/mutual-funds/on_status/on_status-sip-new-account-with-kyc-f2submitted.json

```diff
@@ -189,7 +189,7 @@
                   "id": "form_2"
               },
               "form_response": {
-                "status": "SUCCESS",
```

Comment:
Any specific reason we need this different from SUBMITTED instead of SUCCESS?

Reply:
basically i am treating form submission and form processing as two different steps ... `SUBMITTED` tells that it is successfully submitted and the processing can either fail or succeed, which is done after the next select call is made. To differentiate that having an explicit `SUBMITTED` state is better. Detailed notes are captured here - https://github.com/ONDC-Official/ONDC-FIS-Specifications/blob/draft-FIS14-enhancements/api/components/docs/forms-and-redirects.md